### ISSUE TITLE
Consider changing the name of the "info" namespace to "self" or "this"

### ISSUE THREAD

**Issue by sjperkins** [caracal-pipeline/stimela2#304]
Similar to python, it reinforces the idea that the user is interacting with the current step

Other suggestions:

info.label -> self.name
info.suffix -> self.name_suffix

**Comment by o-smirnov** [comment]
Yeah, ``info`` was an ill-considered choice, it's been bothering me too. I do like the ``self`` suggestion.

**Comment by o-smirnov** [comment]
Implemented a ``self`` namepace that's the same as ``info`` for now. Plan is to deprecate ``info`` in future versions (probably with a deprecation warning first?)
